delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewSource.scala

```diff
@@ -0,0 +1,106 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+
+/**
+  * A source for [[CompositeViewValue]].
+  */
+sealed trait CompositeViewSource extends Product with Serializable {
+
```

Comment:
```scala
  def tpe: CompositeViewSourceType
```

And then create that `CompositeViewSourceType` in a similar way as `ElasticSearchViewType`

Probably the same for projections